recipe/conda_build_config.yaml

```diff
@@ -9,7 +9,7 @@ blas_impl_lib:
 
 - libmkl_rt.so             # [linux64]
 - libmkl_rt.dylib          # [osx and x86_64]
-- mkl_rt.dll               # [win64]
+- mkl_rt.1.dll             # [win64]
```

Comment:
@conda-forge/intel_repack
Not sure if was intentional that the dll-name for `mkl_rt` changed sometime between 2020.4 and 2021.2?

Reply:
Seems the repack [recipe](https://github.com/conda-forge/intel_repack-feedstock/blob/master/recipe/meta.yaml) doesn't contain tests for the lib-names...